src/test/java/com/entropyteam/entropay/testcontainerInit/TestcontainersInitializer.java

```diff
@@ -0,0 +1,44 @@
+package com.entropyteam.entropay.testcontainerInit;
+
+import org.flywaydb.core.Flyway;
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.jdbc.datasource.init.ScriptUtils;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+
+import java.sql.Connection;
+
+public class TestcontainersInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
```

Comment:
Did you follow what we did in recon for this? There's additional support for testcontainers in spring boot 3.1+
Please take a look at this:
- https://docs.spring.io/spring-boot/reference/testing/testcontainers.html
- https://maciejwalkowiak.com/blog/testcontainers-reusable-flyway/

So that we are sure that we are up to date